### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

**Comment by colmsnowplow** [comment]
Thanks @oguzhanunlu, will do. I'm still pushing rebases/commits to deal with the RC releases but for final release I'll make sure that's done.

**Comment by colmsnowplow** [comment]
@lhache and @knservis commits still being added to deal with rc release etc., but at this point the code shouldn't change so think it's ready for code review. :)

File: core/lib/core.ts [hunk 1]
```diff
@@ -845,21 +845,29 @@ export function trackerCore(base64: boolean, callback?: (PayloadData) => void) {
 			tstamp?: Timestamp): PayloadData {
 
 			let event_schema = '';
+			let event_data = {};
 			if (schema === 'change_form'){
 				event_schema = 'iglu:com.snowplowanalytics.snowplow/change_form/jsonschema/1-0-0';
+				event_data = {
+					formId: formId,
+					elementId: elementId,
```
Comment (lhache) [inline core/lib/core.ts]: Dunno if the tracker is in ES6 but if it is, in an object, when the key and the values are the same you can do like:
`{
     formId: formId,
     a: a
}`

becomes

`{
     formId,
     a
}`

Comment (colmsnowplow) [reply core/lib/core.ts]: Cheers Louis, that's cleaner alright. I had hoped to get to this before my time off but am short of time - I'll address this before releasing. Appreciate the feedback!

File: tests/integration/integration.js [hunk 2]
```diff
@@ -315,6 +327,33 @@ define([
...
+
+		'Check a GDPR context': function () {
+      assert.isTrue(checkExistenceOfExpectedQuerystring({
+        cx: function (cx) {
+          var contexts = JSON.parse(decodeBase64(cx)).data;
+          return 1 === lodash.size(
+            lodash.filter(contexts,
+              lodash.overSome(
+                lodash.matches({
+									schema: "iglu:com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0",
```
Comment (knservis) [inline tests/integration/integration.js]: indent?

**Comment by colmsnowplow** [comment]
@lhache I made that improvement you suggested and am QAing a new RC. Once you get the chance let me know if you reckon it's ready to approve now. :)

**Pull request merged**